src/System.Private.CoreLib/shared/System/Version.cs

```diff
@@ -422,31 +422,31 @@ private static bool TryParseComponent(ReadOnlySpan<char> component, string compo
             return ReferenceEquals(v2, v1) ? true : v2.Equals(v1);
         }
 
-        public static bool operator !=(Version v1, Version v2)
+        public static bool operator !=(Version? v1, Version? v2)
         {
             return !(v1 == v2);
         }
 
-        public static bool operator <(Version v1, Version v2)
+        public static bool operator <(Version? v1, Version? v2)
```

Comment:
Why is `v1` nullable? it throws if it is null.

Reply:
Changed it, but it's really strange that:
```C#
new Version(1, 0) < null
```
returns false, but
```C#
new Version(1, 0) > null
```
throws.

Reply:
Actually, this looks like a bug to me.  I'm going to fix it.